velox/functions/prestosql/aggregates/tests/ArrayAggTest.cpp

```diff
@@ -53,17 +61,21 @@ TEST_F(ArrayAggTest, groupBy) {
     // expected result is that there is, for each key, an array of 100
     // elements with, for key k, batch[k[, batch[k + 10], ... batch[k +
     // 90], repeated 10 times.
-    batches.push_back(std::static_pointer_cast<RowVector>(
-        velox::test::BatchMaker::createBatch(
-            ROW({"c0", "a"}, {INTEGER(), ARRAY(VARCHAR())}), 100, *pool_)));
+    batches.push_back(
+        fuzzFlatBatch(ROW({"c0", "a"}, {INTEGER(), ARRAY(VARCHAR())})));
```

Comment:
Would you remove include BatchMaker.h as it is no longer used.

Reply:
Removed